dataland-frontend/tests/e2e/utils/EuTaxonomyFinancialsUpload.ts

```diff
@@ -27,27 +33,101 @@ export function submitEuTaxonomyFinancialsUploadForm(): Cypress.Chainable {
  * @param data the data to fill the form with
  */
 export function fillEuTaxonomyForFinancialsUploadForm(data: EuTaxonomyDataForFinancials): void {
-  cy.get(`input[name="reportingPeriod"]`).type("2023");
+  cy.get('button[data-test="upload-files-button"]').click();
+  cy.get("input[type=file]").selectFile("tests/e2e/fixtures/pdfTest.pdf", { force: true });
+  cy.get('div[data-test="uploaded-files"]')
+    .should("exist")
+    .find('[data-test="uploaded-files-title"]')
+    .should("contain", "pdf");
```

Comment:
Since you already know the file name by referencing it via the path in line 37, you could also check for it:
`.should("contain", "pdfTest.pdf");`

Reply:
Or even better. Smth like:
`const testFileName = "pdfTest.pdf"
cy.get("input[type=file]").selectFile("tests/e2e/fixtures/${testFileName}", { force: true });
...
.should("contain", ${testFileName);
`